components/button-cards/_category-card/category-full-row.hbs

```diff
@@ -0,0 +1,111 @@
+<div class="container">
+    <div class="row card-row">
+        <div class="col-12 col-md-4">
+            <div class="msds-category-card">
+                <a class="msds-category-card__link" href="#" alt="msds-category-cards">
+                    <div class="msds-category-card__header">
+                        <div class="msds-category-card__header-body">Lorem, ipsum dolor</div>
+                        <div class="msds-category-card__header-icon-container">
+                            <svg class="msds-category-card__header-icon">
+                                <use href="{{ path '/msds-spritemap.svg' }}#right-arrow" />
+                            </svg>
+                        </div>
+                    </div>
+                    <div class="msds-category-card__content">
+                        <div class="msds-category-card__content-body">Lorem ipsum dolor sit amet</div>
+                    </div>
+                </a>
+            </div>
+        </div>
+
+        <div class="col-12 col-md-4">
+            <div class="msds-category-card">
+                <a class="msds-category-card__link" href="#" alt="msds-category-cards">
+                    <div class="msds-category-card__header">
+                        <div class="msds-category-card__header-body">Lorem ipsum dolor sit amet consectetur</div>
+                        <div class="msds-category-card__header-icon-container">
+                            <svg class="msds-category-card__header-icon">
+                                <use href="{{ path '/msds-spritemap.svg' }}#right-arrow" />
+                            </svg>
+                        </div>
+                    </div>
+                    <div class="msds-category-card__content">
+                        <div class="msds-category-card__content-body">Lorem ipsum dolor sit amet consectetur adipisicing elit. Neque</div>
+                    </div>
+                </a>
```

Comment:
What are your thoughts on the BEM convention here?

Reply:
I would say the category cards are fine, they have been approved. This task was focused on the tabs, if we need to work on the category cards some more we need a task (This file was only created for a placeholder in the tabs content)

Reply:
In my opinion everything is fine there. Danny what is a problem, are you drunk?